app/templates/docs/enterprise.html

```diff
@@ -0,0 +1,625 @@
+                </li>
+                <li><code class="docutils literal"><span class="pre">idOrganizations</span></code></li>
+                <li><code class="docutils literal"><span class="pre">products</span></code></li>
+                <li><code class="docutils literal"><span class="pre">userTypes</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idMembers</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idOrganizations</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li><code class="docutils literal"><span class="pre">members</span></code> (optional)
+          <ul>
+            <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">none</span></code></li>
+            <li><strong>Valid Values:</strong>
+              <ul>
+                <li><code class="docutils literal"><span class="pre">none</span></code></li>
+                <li><code class="docutils literal"><span class="pre">normal</span></code></li>
+                <li><code class="docutils literal"><span class="pre">admins</span></code></li>
+                <li><code class="docutils literal"><span class="pre">owners</span></code></li>
+                <li><code class="docutils literal"><span class="pre">all</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li><code class="docutils literal"><span class="pre">member_fields</span></code> (optional)
+          <ul>
+            <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">avatarHash, fullName, initials, username</span></code></li>
+            <li><strong>Valid Values:</strong>
+              <ul>
+                <li><code class="docutils literal"><span class="pre">avatarHash</span></code></li>
+                <li><code class="docutils literal"><span class="pre">fullName</span></code></li>
+                <li><code class="docutils literal"><span class="pre">initials</span></code></li>
+                <li><code class="docutils literal"><span class="pre">username</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li><code class="docutils literal"><span class="pre">member_filter</span></code> (optional)
+          <ul>
+            <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">none</span></code></li>
+            <li>
+              <strong>Valid Values:</strong>
+              <p>Pass a <a href="/advanced-reference/scim#filtering">SCIM-style query</a> to filter members. This takes precedence over the all/normal/admins value of <code>members</code>. If any of the below member_* args are set, the member array will be paginated.</p>
```

Comment:
The “below” doesn't make sense now this is a note on the `member_filter` parameter.

> If any of the `member_*` args are set, the member array will be paginated.